### DISCUSSION TITLE
Mac OS problems

### DISCUSSION THREAD

**Discussion by aj-arts** [pystardust/ani-cli#185] [Q&A] [answered]
I am newbie and saw this cool video about this and wanted to try it but the file is an executable and doesn't seem to be working on Mac. I don't have any idea what to do or whether this software works on Mac? can you tell me what am I doing wrong bcoz every time I execute ani-cli <search> command it shows 
/usr/local/bin/ani-cli: line 150: /Users/ajinkyagokule/.cache/ani-hsts: No such file or directory 
or if this is only linux exclusive?

**Comment by tennyson-mccalla** [comment]
Just create that file (i.e. /Users/ajinkyagokule/.cache/ani-hsts).

I've gotten it to work fine on my Mac. Just be sure you have mpv and take care of the line calls setsid -f. Have a look in the pull requests for Mac.

**Reply by aj-arts** [reply]
I did what you told me. I have mpv and when I enter the episode no and press enter nothing pops up in mpv.

**Reply by tennyson-mccalla** [reply]
And did you check the difference between what you have and the pull request I hinted at?
https://github.com/pystardust/ani-cli/pull/137

**Reply by aj-arts** [reply]
lul I restarted  my Mac and it worked thnx for the info......it honestly very convenient thank you so much

**Reply by tennyson-mccalla** [reply]
NP. Glad it worked out. This is an awesome project.

**Comment by aj-arts** [comment]
Ohk I will check it out...thanks for the info

**Comment by port19x** [comment, marked as answer]
There are plans to bring full support to both mac and linux.
On mac, there is a high chance that PR #137 will be enough to make everything work
On windows, #73 will need to be fixed.
In the somewhat distant future, real packages for mac (brew) and windows (exe) could also be desireable
